src/Identity/Extensions.Core/src/IdentityBuilder.cs

```diff
@@ -70,6 +72,7 @@ private IdentityBuilder AddScoped(Type serviceType, [DynamicallyAccessedMembers(
     /// </summary>
     /// <typeparam name="TValidator">The user validator type.</typeparam>
     /// <returns>The current <see cref="IdentityBuilder"/> instance.</returns>
+    [UnconditionalSuppressMessage("AOT", "IL3050", Justification = "MakeGenericType is safe because generic type and user type are reference types.")]
```

Comment:
What does "generic type" in 

> because generic type and user type are reference types

mean? Should it just be

```suggestion
    [UnconditionalSuppressMessage("AOT", "IL3050", Justification = "MakeGenericType is safe because UserType is a reference type.")]
```